roscala/src/main/scala/coop/rchain/rosette/Monitor.scala

```diff
@@ -6,9 +6,9 @@ case class Monitor(id: Ob,
                    obCounts: Long,
                    tracing: Boolean)
     extends Ob {
-  def reset(): Unit = {}
-  def start(): Unit = timer.start()
-  def stop(): Unit = timer.stop()
+  def reset(): Unit                  = {}
```

Comment:
Do we really want such alignment for methods?

Reply:
@DoSOfRedRiver it really helps to skim through the code especially with tons of those small, tiny methods.